ginivision/src/doc/source/supporting-tablets.rst

```diff
@@ -0,0 +1,63 @@
+Supporting Tablets
+==================
+
+The Gini Vision Library can be used on tablets, too. Some UI elements have been adapted to offer the best user experience for tablet users. Requirements and resources were also adapted.
+
+You may skip to the `Quick Checklist`_ to get an overview of the steps required for supporting tablets.
+
+Extraction Quality Considerations
+---------------------------------
+
+Tablets have generally lower camera resolutions and some (even popular ones) lack an LED flash. Please note that you may experience worse extraction quality on tablets compared to phones.
+
+Hardware Requirements
+---------------------
+
+We disabled the camera flash requirement for tablets. Camera flash is not a standard feature for tablets and even some popular models like the Samsung Galaxy Tab S2 don't have an LED flash.
```

Comment:
...a Led flash.

Reply:
I guess it depends on how you read it. I read it as eLEDe so it's `an LED`. Can you help us out @a-benson?

Reply:
I would leave the "an" in. I'm honestly not sure what is grammatically correct when you have an abbreviation, but it sounds correct with the "an". :)